package/opt/hassbian/suites/files/appdaemon.conf

```diff
@@ -1,7 +1,7 @@
-AppDaemon:
-  logfile: STDOUT
-  errorfile: STDERR
-  log_generations: 3
+appdaemon:
   threads: 10
-HASS:
-  ha_url: http://127.0.0.1:8123
\ No newline at end of file
+  plugins:
+    HASS:
+      type: hass
+      ha_url: http://127.0.0.1:8123
+      ha_key:
```

Comment:
Can you comment out this part?
If you see #142 it still fails if the ha_key: is blank..
`      #ha_key:`

Reply:
I notice the issue. Did it mean that we should add latitude longitude time_zone... conf by default? But I search Appdaemon’s docs and these variables should be optional.

Reply:
I don't think that is necessary, If you change `ha_key:` to `#ha_key:` it will start.
I'm working on setting the password if one is supplied in the script now.